spec/support/acceptance/curl_command.rb

```diff
@@ -0,0 +1,63 @@
+# https://github.com/mizzy/serverspec/pull/611 was rejected so adding it here.
```

Comment:
One of the rejected serverspec PRs had the recommendation to use https://github.com/ryotarai/infrataster for this instead

Might be worth checking out longer term

Reply:
That repo hasn't seen activity since 2017. That concerns me.